beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -3205,6 +3210,9 @@ async fn deneb_prune_blobs_no_finalization() {
         Some(deneb_fork_slot)
     );
     check_blob_existence(&harness, Slot::new(0), harness.head_slot(), true);
```

Comment:
You might have already noticed, this test [runs multiple times](https://github.com/sigp/lighthouse/blob/166f6df5a973e5edd6faf17812f483260f50d2a1/Makefile#L145-L146 ) on CI for all forks.

The code that reads `FORK_NAME` from env var is here:
https://github.com/sigp/lighthouse/blob/6f31d44343086ddaa70b396632c0eee724256454/beacon_node/beacon_chain/src/test_utils.rs#L193-L206

So this assertion would fail when `FORK_NAME=fulu`, because we don't store any blobs from since the Fulu fork. I think it might be cleaner to start a separate test that verifies data column pruning instead of having to add conditions on every assertion.

Reply:
Yes, that's one of the reasons - but also it's no longer useful to the CL from Fulu.
- The network form of blobs on the beacon chain is `DataColumnSidecar`, we'll stop using `BlobSidecar` on p2p from Fulu slots.
- Another reason is fullnodes will no longer have blobs, but just samples (DataColumns)